### PR TITLE
Add missing Permissions to make a dataset public

### PR DESCRIPTION
**Pull request by samuel-kerrien**

### CODE REVIEW

File: nexuscli/acls.py [hunk 1]
```diff
@@ -13,6 +13,9 @@
           }
 
 _PROJECTS_READ_ = "projects/read"
+_SCHEMAS_READ_ = "schemas/read"
+_RESOURCES_READ_ = "resources/read"
+_VIEWS_READ_ = "views/read"
```
Comment (bogdanromanx) [inline nexuscli/acls.py]: `views/query` here

File: nexuscli/acls.py [hunk 2]
```diff
@@ -13,6 +13,9 @@
           }
 
 _PROJECTS_READ_ = "projects/read"
+_SCHEMAS_READ_ = "schemas/read"
```
Comment (bogdanromanx) [inline nexuscli/acls.py]: remove

**Review by bogdanromanx — CHANGES_REQUESTED** [review]

File: nexuscli/acls.py (hunk 2 above)
Comment (bogdanromanx) [reply nexuscli/acls.py]: this permission doesn't exist

File: nexuscli/acls.py (hunk 1 above)
Comment (samuel-kerrien) [reply nexuscli/acls.py]: ok

File: nexuscli/acls.py (hunk 2 above)
Comment (samuel-kerrien) [reply nexuscli/acls.py]: why would you not want to allow users to read schemas ?

Comment (bogdanromanx) [reply nexuscli/acls.py]: we've decided to use `resources/read` instead for all resources (to simplify the implementation); there's is no such permission defined or used in the system. If you can `resources/read` you can read schemas.

**Review by bogdanromanx — APPROVED** [review]

**Pull request merged**